assembl/views/__init__.py

```diff
@@ -226,10 +282,25 @@ def process_export_list(ls):
 
     analytics_url = config.get('web_analytics_piwik_url', None)
 
+    get_route = create_get_route(request, discussion)
+    providers = get_provider_data(get_route)
+
+    errors = request.session.pop_flash()
+    if kwargs.get('error', None):
+        errors.append(kwargs['error'])
+    if errors:
+        kwargs['error'] = '<br />'.join(errors)
+    messages = request.session.pop_flash('message')
+    if messages:
+        kwargs['messages'] = '<br />'.join(messages)
+
+    admin_email = config.get('assembl.admin_email') or "assembl.admin@bluenove.com"
```

Comment:
It's a fallback to the config file. The alternative is to refuse to start assembl, saying that it is not properly configured... which may be a good idea, actually. Other opinions?